### PR TITLE
#3655 BCSC Info Panel

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:*
https://github.com/bcgov/entity/issues/3655

*Description of changes:*

* Implements a static testimonial section ( Placeholder: Currently pending more design to transition to dynamic )

* Implements the BCSC Info Panel

* Style and Image Updates.

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [x] No lint errors
- [x] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/BCSCPanel.vue [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<template>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BCSCPanel.vue:1]: I suggest camel-case for the filename (ie, BcscPanel.vue).

Comment (cameron-freshworks) [reply auth-web/src/components/auth/BCSCPanel.vue:1]: ah definitely. Will do

File: auth-web/src/components/auth/BCSCPanel.vue [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  <v-container id="bcsc-container" class="my-10">
+    <h2 class="mt-5">Create an Account to Incorporate</h2>
+    <p class="my-5">Log in securely using your mobile BC Services Card, government's trust way to access online services</p>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BCSCPanel.vue:4]: trust -> trusted

Comment (cameron-freshworks) [reply auth-web/src/components/auth/BCSCPanel.vue:4]: Updated in next commit!

File: auth-web/src/components/auth/BCSCPanel.vue [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <template v-if="!userProfile">
+          <v-btn large color="#fcba19" @click="emitLogin()" class="mt-5">
+            Log in with BC Services Card
+          </v-btn>
+          <p class="my-5">New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
+            <u>Create a BC Registries Account</u></a>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BCSCPanel.vue:37]: Does anchor text still need to be underlined explicitly? I thought that was a default browser style (and as such, override-able by user styling).

File: auth-web/src/components/auth/BCSCPanel.vue [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  }
+})
+export default class BCSCPanel extends Vue {
+  private secureBulletPoints: Array<any> = [
+    { text: 'A mobile card is a representation of your BC Services Card on your mobile device. It\'s used to prove who you are when you log in to access government services online.' },
+    { text: 'Only your name and a unique identifier is stored on the mobile device.' }
+  ]
+
+  private easeBulletPoints: Array<any> = [
+    { text: 'It normally takes about 5 minutes to set up a mobile card.' },
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BCSCPanel.vue:62]: Lies! I started my setup in Aug 2019 and I still haven't finished it 😛 

(Ignore this comment, obviously.)

File: auth-web/src/components/auth/BCSCPanel.vue [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    { text: 'you can verify your identity by video right from your mobile device. You don\'t need to go in person unless you can\'t verify by video.' }
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BCSCPanel.vue:63]: you -> You

File: auth-web/src/components/auth/BCSCPanel.vue [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  ]
+
+  @Prop() userProfile
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BCSCPanel.vue:66]: Please follow other code style:
- property on next line
- private
- type?
- default?

File: auth-web/src/components/auth/TestimonialQuotes.vue [hunk 7]
```diff
@@ -0,0 +1,49 @@
+<template>
+  <section id="testimonials-container">
+    <v-container class="quote-container">
+      <p>"We're incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values
+        and the role we want to Play."</p>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/TestimonialQuotes.vue:5]: Play -> play

File: auth-web/src/components/auth/TestimonialQuotes.vue [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+      <p><strong>- BC Benefit Company user</strong></p>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/TestimonialQuotes.vue:6]: You might try `&ndash;` or `&hyphen;` and see if it looks any better than `-`.

File: auth-web/src/components/auth/BCSCPanel.vue (hunk 3 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/BCSCPanel.vue:37]: This was a good question! Your right, by default it is underlined and it took me a few mins to find why this wasn't, which was due to the lack of a specified url, `href`.

File: auth-web/src/components/auth/BCSCPanel.vue (hunk 5 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/BCSCPanel.vue:63]: 🤦

File: auth-web/src/components/auth/BCSCPanel.vue (hunk 6 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/BCSCPanel.vue:66]: fixed in next commit!

File: auth-web/src/components/auth/TestimonialQuotes.vue (hunk 8 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/TestimonialQuotes.vue:6]: `&ndash;` looks best i think, thanks!

File: auth-web/src/components/auth/BCSCPanel.vue (hunk 3 above)
Comment (severinbeauvais) [reply auth-web/src/components/auth/BCSCPanel.vue]: Great. For others: my concern here that hard-coding this style does not allow the user agent CSS to override it.

File: auth-web/src/components/auth/BcscPanel.vue [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <template v-if="!userProfile">
+          <v-btn large color="#fcba19" @click="emitLogin()" class="mt-5">
+            Log in with BC Services Card
+          </v-btn>
+          <p class="my-5">New to BC Registries?
+            <a @click="emitAccountDialog()" class="create-account-link"><u>Create a BC Registries Account</u></a>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/BcscPanel.vue:39]: Hard-coded underline here too.

File: auth-web/src/components/auth/BCSCPanel.vue (hunk 1 above)
Comment (severinbeauvais) [reply auth-web/src/components/auth/BCSCPanel.vue]: Haha, I meant everywhere it is used. But not worth a lot of time on this.

File: auth-web/src/components/auth/TestimonialQuotes.vue (hunk 7 above)
Comment (severinbeauvais) [reply auth-web/src/components/auth/TestimonialQuotes.vue]: Fixed.

File: auth-web/src/components/auth/BCSCPanel.vue (hunk 1 above)
Comment (cameron-freshworks) [reply auth-web/src/components/auth/BCSCPanel.vue]: Oops, sorry. Missed a few. Fix coming in next commit.

**Pull request merged**